types/node-pixel/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for node-pixel 0.11
```

Comment:
You've created this package using the old method; you may want to retry using a later version of dts-gen or manually fix this up.

Reply:
The new way looks really great! It's way easier to understand, I've updated things, thanks.